docs/csharp/versioning.md

```diff
@@ -43,8 +43,9 @@ will have to be updated. This is a huge breaking change and is strongly discoura
 * Method signatures: When updating a method behaviour requires you to change its signature as well, you should instead create an overload so that code calling into that method will still work.
 You can always manipulate the old method signature to call into the new method signature so that implementation remains consistent.
 * [Obsolete attribute](programming-guide/concepts/attributes/common-attributes.md#Obsolete): You can use this attribute in your code to specify classes or class members that are deprecated and likely to be removed in future versions.
-This ensures developers utilizing your library are better prepared for breaking changes.
+  This ensures developers utilizing your library are better prepared for breaking changes.
```

Comment:
I don't understand the extra spaces here

```suggestion
This ensures developers utilizing your library are better prepared for breaking changes.
```

Reply:
Because it's part of the list item before it, and there is another list item after it. It could also just be pulled up into the previous line

Reply:
That would be better I think.

Reply:
done